worker/games.py

```diff
@@ -61,6 +61,34 @@ def github_api(repo):
 def enc(s):
   return s.encode('utf-8')
 
+def required_net(engine):
+  net = None
+  print('Obtaining EvalFile of %s ...' % (os.path.basename(engine)))
+  p = subprocess.Popen([engine, 'uci'], stdout=subprocess.PIPE, universal_newlines=True, bufsize=1, close_fds=not IS_WINDOWS)
+
+  for line in iter(p.stdout.readline,''):
+    if 'EvalFile' in line:
+      net = line.split(' ')[6].strip()
+
+  p.wait()
+  p.stderr.close()
```

Comment:
`p.stderr` was not piped, so here you need `p.stdout.close()`

Reply:
thanks!